public/src/components/Members.tsx

```diff
@@ -52,27 +52,35 @@ const Members = (): JSX.Element => {
 
         if (isPending(enrollment)) {
             data.push(
-                <div>
-                    <i className="badge badge-primary" style={{ cursor: "pointer" }}
-                        onClick={() => { actions.updateEnrollment({ enrollment, status: Enrollment_UserStatus.STUDENT }) }}>
-                        Accept
-                    </i>
-                    <i className="badge badge-danger clickable ml-1"
-                        onClick={() => actions.updateEnrollment({ enrollment, status: Enrollment_UserStatus.NONE })}>
-                        Reject
-                    </i>
+                <div className="d-flex">
+                    <DynamicButton onClick={() => actions.updateEnrollment({ enrollment, status: Enrollment_UserStatus.STUDENT })}
+                        className="mr-2"
```

Comment:
This is nice; more readable.

Reply:
should the `onClick` be on a separate line too, to make it consistent with the `DynamicButton` below on Lines 71 and 78.